app/routes/dashboard/exercise-add.tsx

```diff
@@ -1,7 +1,7 @@
-import { createFileRoute, useNavigate } from "@tanstack/react-router";
-import { useForm } from "react-hook-form";
-import { z } from "zod";
-import { zodResolver } from "@hookform/resolvers/zod";
+import { createFileRoute, useNavigate } from '@tanstack/react-router'
```

Comment:
Sorry, for the changes to single quotes, my editor didn't pickup a prettier config file 🤷🏼‍♂️